src/sql/src/catalog.rs

```diff
@@ -220,6 +222,7 @@ impl fmt::Display for CatalogItemType {
             CatalogItemType::Sink => f.write_str("sink"),
             CatalogItemType::View => f.write_str("view"),
             CatalogItemType::Index => f.write_str("index"),
+            CatalogItemType::Type => f.write_str("user-defined type"),
```

Comment:
Let's just say "type" here and above. I think we're likely to have system types in here eventually.